gcloud.go

```diff
@@ -32,7 +32,7 @@ func NewGCloudClient(projectId string, zone string) (gcloud *GCloud, err error)
 	client, err := google.DefaultClient(gcloud.Context, compute.ComputeScope)
 
 	if err != nil {
-		err = fmt.Errorf("Error creating compute client: %v", err)
+		err = fmt.Errorf("Error creating compute client:\n%v", err)
```

Comment:
I wonder what's best, because the function return an error which is then displayed via zerolog in the main.go.